src/state/KindeProvider.tsx

```diff
@@ -23,7 +24,7 @@ type KindeProviderProps = {
   isDangerouslyUseLocalStorage?: boolean;
   logoutUri?: string;
   redirectUri: string;
-  onRedirectCallback?: () => void;
+  onRedirectCallback?: (user: KindeUser, state?: State) => void;
```

Comment:
@stephenkao thanks so much for the PR :) 

In this case the second argument isn't the `State` type that contains `isLoading`, `user` etc it is actually the application state before the redirect took place, so it is a consumer defined object. This [doc](https://kinde.com/docs/developer-tools/react-sdk/#persisting-application-state) explains how it works.

Would you mind updating the signature in this PR to reflect this?

Otherwise looks great - thanks again :)

Reply:
Ah, thanks for catching this!  I've updated it [in this commit](https://github.com/kinde-oss/kinde-auth-react/pull/26/commits/f154c7c3ccab4031f9031595d5e2d4a23abfb87d) to accept an optional generic object to account for this.  Let me know if that's what you were thinking or if you had something else in mind!